tasks/build_tags.py

```diff
@@ -15,6 +15,7 @@
     "docker",
     "ec2",
     "etcd",
+    "fargateprocess",
```

Comment:
Do we want to build with fargate on the core agent ?

Reply:
no, that's why it's in `PROCESS_ONLY_TAGS` as well

Reply:
Why do we have this logic of adding process only tags to ALL_TAGS and then removing them instead of having all common tags in ALL_TAGS and adding PROCESS_ONLY_TAGS to the list when building process ?

Reply:
TBH I don't have the answer, maybe @DataDog/processes  and @DataDog/agent-platform have more context?
In this PR I followed the existing pattern, `PROCESS_ONLY_TAGS` was there. And I believe it's not in the scope of this particular PR to change that even though I do agree that adding instead of removing would be simpler.

Reply:
Yes, I wanted to include the `orchestrator` tag only in the process-agent because it was raising significantly the dsd binary size, I might have been too quick on fixing this though, open to any improvement 👍  

I added it to `ALL_TAGS` following this comment https://github.com/DataDog/datadog-agent/blob/8c6bc642a43345a5ca67e892356e18bb8484a18b/tasks/build_tags.py#L8 but we should probably revisit the logic

Reply:
thanks Pierre for clarifying this, @julien-lebot can you approve this PR on behalf of Platform and take care of revamping this later? I believe your team owns the `tasks` directory